### ISSUE TITLE
Feature req: Option to exclude certain data bags

### ISSUE THREAD

**Issue by jjhuff** [bmarini/knife-inspect#41]
I have a data bag that is dynamically updated, so i don't keep it in git.

I'd like to be able to check all data bags except that one.

Thanks!

**Comment by gregkare** [comment]
Hi. I'm open to a pull request for this, I won't implement it myself though

**Comment by jjhuff** [comment]
Yup, I understand. I'll try to get a PR out in a couple days. Thanks!
